util/setup_dev.sh

```diff
@@ -136,6 +136,30 @@ EOF
          use_http2
          use_http3
          env_to_nginx
+EOF
+    patch --forward --ignore-whitespace lib/perl5/Test/Nginx/Util.pm <<'EOF'
+    @@ -977,6 +977,11 @@
+         my $post_main_config = $block->post_main_config;
+         my $err_log_file = $block->error_log_file;
+         my $server_name = $block->server_name;
+    +    my $workers = $block->workers;
+    +
+    +    if (!$workers) {
+    +        $workers = $Workers;
+    +    }
```

Comment:
@casimiro Is this patch truly necessary? Ideally we only include fixes in the patches of `setup_dev.sh` and as little behavioral changes as necessary. What does this patch really do? Isn't the default number of workers already 1?

Reply:
Oh I see, it seems to be so that individual tests can influence the number of workers with `--- workers`, correct?